Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:alpine
+WORKDIR '/root'
+RUN apk add ruby
+RUN apk add ruby-dev
+RUN apk add git
+ENV BUNDLER_VERSION 2.0.1
+RUN gem install --no-rdoc --no-ri bundler
```

Comment:
You can skip installing bundler if the Ruby was 2.6+ since the command has been bundled in Ruby from version 2.6.